cmake/PackageConfig.cmake.in

```diff
@@ -8,3 +8,6 @@ include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
 get_target_property(@PROJECT_NAME@_BUILD_TYPES @PROJECT_NAME@::@PROJECT_NAME@ IMPORTED_CONFIGURATIONS)
 
 check_required_components("@PROJECT_NAME@")
+
+get_target_property(location @PROJECT_NAME@::@PROJECT_NAME@ LOCATION)
+message(STATUS "Found @PROJECT_NAME@: ${location} (found version \"@PROJECT_VERSION@\")")
```

Comment:
I don't think the version bit works.

For `version`, one needs to include 
```
include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-config-version.cmake")
```

which sets the variable `PACKAGE_VERSION`

Together to be used like so:
```
include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-config-version.cmake")
message(STATUS "Found @PROJECT_NAME@ VERSION  ${PACKAGE_VERSION}")
```

Reply:
actually, it does work. nevermind.